scripts/mc_log_autocorrelations.py

```diff
@@ -9,78 +9,142 @@
 """
 from __future__ import absolute_import, division, print_function
 
-import datetime
+import socket
+import time
+from builtins import int
+
+import re
+from astropy.time import Time, TimeDelta
 import numpy as np
 import redis
-import sys
 
-import six.moves as sm
 
 from hera_mc import autocorrelations, mc
 
 
-# Yay hardcoded config settings:
+def test_redis_connection(rsession, redispool, max_tries=3):
+    redis_count = 0
+    while redis_count <= max_tries:
+        try:
+            rsession.ping()
+            # got a good connection return to the monitor
+            return rsession
+        except redis.exceptions.ConnectionError:
+            print("Redis connection not found, attempting to reconnect.")
+            rsession = redis.Redis(connection_pool=redis_pool)
+            redis_count += 1
+            if redis_count == max_tries:
+                raise
 
-redis_host = 'redishost'
 
-antnums = list(sm.range(128))
+# Preliminaries. We have a small validity check since the M&C design specifies
+# the memory, network, and system load are to be 5-minute averages.
```

Comment:
there's no memory, network or system load in this script as far as I can tell

Reply:
ah yes good call. That was a copy/pasta mistake.